rfc/src/mdn.mdown

```diff
@@ -10,6 +10,12 @@
 
     date = 2020-07-27T00:00:00Z
 
+    [seriesInfo]
+    name="Internet-Draft"
+    value="draft-jenkins-mdn-00"
```

Comment:
This should be `draft-ietf-jmap-mdn` – it was [adopted as an official doc](https://tools.ietf.org/html/draft-ietf-jmap-mdn) a while back.

Reply:
That was probably a copy-paste error. Sorry. Fixed it.